app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt

```diff
@@ -428,18 +311,11 @@ class AdministratorControlsActivityTest {
       )
     ).use {
       testCoroutineDispatchers.runCurrent()
-      onView(withId(R.id.administrator_controls_list)).perform(
-        scrollToPosition<RecyclerView.ViewHolder>(
-          4
-        )
-      )
+      scrollToPosition(position = 4)
       onView(withId(R.id.log_out_text_view)).perform(click())
-      onView(withText(R.string.log_out_dialog_message)).inRoot(isDialog())
-        .check(matches(isDisplayed()))
-      onView(withText(R.string.log_out_dialog_okay_button)).inRoot(isDialog())
-        .check(matches(isDisplayed()))
-      onView(withText(R.string.log_out_dialog_cancel_button)).inRoot(isDialog())
-        .check(matches(isDisplayed()))
+      verifyTextInDialog(textInDialog = context.getString(R.string.log_out_dialog_message))
```

Comment:
If we only pass in a string, you could also change verifyTextInDialog to take a ``@StringRes`` string resource ID and retrieve the string from the context in the helper.

Reply:
done